packages/itwin/map-layers/src/ui/widget/SubLayersPopupButton.tsx

```diff
@@ -29,39 +25,33 @@ export function SubLayersPopupButton(props: SubLayersPopupButtonProps) {
     setPopupOpen(!popupOpen);
   }, [popupOpen]);
 
-  const onOutsideClick = React.useCallback(() => {
-    setPopupOpen(false);
-  }, []);
-
-  // eslint-disable-next-line @typescript-eslint/no-deprecated
-  const isOutsideEvent = React.useCallback((e: OutsideClickEvent) => {
-    // if clicking on button that open panel - don't trigger outside click processing
-    return !!buttonRef.current && e.target instanceof Node && !buttonRef.current.contains(e.target);
-  }, []);
-
-  // eslint-disable-next-line @typescript-eslint/no-deprecated
-  const panelRef = useOnOutsideClick<HTMLDivElement>(onOutsideClick, isOutsideEvent);
-
   return (
     <>
-      <Button
-        size="small"
-        styleType="borderless"
-        ref={buttonRef}
-        className="map-manager-item-sub-layer-button"
-        title={popupOpen ? hideSubLayersLabel : showSubLayersLabel}
-        onClick={togglePopup}
-      >
-        <SvgLayers />
-      </Button>
-      {/*eslint-disable-next-line @typescript-eslint/no-deprecated */}
-      <Popup isOpen={popupOpen} position={RelativePosition.BottomRight} onClose={onOutsideClick} target={buttonRef.current}>
-        <div className="map-transparency-popup-panel">
-          <div ref={panelRef} className="map-manager-sublayer-panel">
-            <SubLayersPanel {...props} width={390} height={350} />
+      <Popover
```

Comment:
@aruniverse we previously had a discussion on whether our packages should embed ThemeProvider by default. While I don't know what came out of that, do you think we should add `portal` [property](https://itwinui.bentley.com/docs/popover#props) to the `Popover` components?

Reply:
dont think portal is neeeded, nor should we wrap our component in theme provider